wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-# WG Manifests Charter
+# WG Platform/Manifests Charter
 
-This charter adheres to the conventions, roles and organization management
-outlined in [wg-governance].
+This platform/manifests charter describes the working mode / reality / status quo of the last 5 years as of March 2025.
+It tries to be as lean as possible and balance community and commercial interests.
 
 ## Scope
 
-- Provide a catalog (centralized repository) of Kubeflow application manifests.
-- Provide a catalog of third-party apps for common services.
-
-### In scope
-
-#### Code, Binaries and Services
-
-- Maintain tooling to automate copying manifests from upstream app repos.
```

Comment:
This is still definitely necessary.

Reply:
Yes and covered by "Synchronize the manifests (Helm, Kustomize) between working groups"